LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Microsoft Corporation
+Copyright (c) 2015 Microsoft. All rights reserved.
```

Comment:
Hey @davidchesnut. Just making sure. Is it *Microsoft* or *Microsoft Corporation*